static/templyte/src/app/my-templates/my-templates.component.html

```diff
@@ -22,14 +22,16 @@
 </div>
 
 <div *ngFor="let template of templateList | async">
-    <mat-card class="template-card" (click)="openTemplate(template.uid)">
+    <mat-card class="template-card">
         <p class="template-title">{{template.templateName}}</p>
         <p>by {{template.authorName}}</p>
         <mat-chip-list>
             <mat-chip *ngFor="let tag of template.tags">
                 {{tag.name}}
             </mat-chip>
         </mat-chip-list>
+        <button mat-raised-button (click)="openTemplate(template.uid)">Download</button>
+        <button mat-raised-button (click)="deleteTemplate(template.uid)">Delete</button>
```

Comment:
@CaylaVinzons can you look into how you want to style this? (this can also be done in a later PR)